sdk/core/core-paging/CHANGELOG.md

```diff
@@ -1,13 +1,9 @@
 # Release History
 
-## 1.3.0 (Unreleased)
+## 1.3.0 (2022-05-18)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
 ### Other Changes
```

Comment:
```suggestion
```
Push it to `features added` instead?

Reply:
I feel like it is adding a missing behavior rather than adding a new feature but I don’t have strong feelings about it. Let me know if you feel strongly about it.

Reply:
Only concern was that there are no other changes to call this as "Other".

I don't really care though, it's fine.